{
	"id": "CVE-2023-6839",
	"sourceIdentifier": "ed10eef1-636d-4fbe-9993-6890dfa878f8",
	"vulnStatus": "Undergoing Analysis",
	"published": "2023-12-15T11:15:48.003",
	"lastModified": "2023-12-15T13:41:51.403",
	"descriptions": [
		{
			"lang": "en",
			"value": "Due to improper error handling, a REST API resource could expose a server side error containing an internal WSO2 specific package name in the HTTP response.\n\n"
		},
		{
			"lang": "es",
			"value": "Debido a un manejo inadecuado de errores, un recurso de API REST podría exponer un error del lado del servidor que contenga un nombre de paquete interno específico de WSO2 en la respuesta HTTP."
		}
	],
	"references": [
		{
			"source": "ed10eef1-636d-4fbe-9993-6890dfa878f8",
			"url": "https://security.docs.wso2.com/en/latest/security-announcements/security-advisories/2022/WSO2-2021-1334/"
		}
	],
	"metrics": {
		"cvssMetricV31": [
			{
				"source": "ed10eef1-636d-4fbe-9993-6890dfa878f8",
				"type": "Secondary",
				"cvssData": {
					"version": "3.1",
					"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
					"attackVector": "NETWORK",
					"attackComplexity": "LOW",
					"privilegesRequired": "NONE",
					"userInteraction": "NONE",
					"scope": "UNCHANGED",
					"confidentialityImpact": "LOW",
					"integrityImpact": "NONE",
					"availabilityImpact": "NONE",
					"baseScore": 5.3,
					"baseSeverity": "MEDIUM"
				},
				"exploitabilityScore": 3.9,
				"impactScore": 1.4
			}
		]
	},
	"weaknesses": [
		{
			"source": "ed10eef1-636d-4fbe-9993-6890dfa878f8",
			"type": "Secondary",
			"description": [
				{
					"lang": "en",
					"value": "CWE-209"
				}
			]
		}
	]
}
